tests/loader_wsi_tests.cpp

```diff
@@ -580,4 +700,44 @@ TEST_F(WsiTests, GetPhysicalDeviceWaylandPresentICDSupport) {
 
     ASSERT_EQ(VK_TRUE, env->vulkan_functions.vkGetPhysicalDeviceWaylandPresentationSupportKHR(physical_device, 0, nullptr));
 }
-#endif
\ No newline at end of file
+
+TEST_F(WsiTests, WaylandGetPhysicalDeviceSurfaceSupportKHR) {
+    FrameworkEnvironment env{};
```

Comment:
This is wrong. TEST_F(WsiTests, provides a framework environment, so this is going to cause confusion as two test frameworks are vying for control.

Reply:
So my tests failed if I mixed `TEST_F` with `TEST`, so to resolve this, I converted all the WSI tests to `TEST`.